cypress/locators/grid/index.js

```diff
@@ -7,3 +7,5 @@ export const gridPod = index => cy.get(DLS_ROOT)
   .find('div')
   .find(`div:nth-child(${index})`);
 export const gridComponent = () => cy.get(GRID_COMPONENT);
+export const justifyGridComponent = () => cy.iFrame('#grid-justify');
```

Comment:
move this `#grid-justify` to locators, please

Reply:
fixed